example/controller-registration.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       values:
         image:
-          tag: v0.11.1
+          tag: v0.11.3
```

Comment:
maybe worth a v0.12.0 ?

Reply:
This version is always behind, it's auto-generated by `make generate-in-docker`  and it does not know about a future release tag. It is also not really necessary because in a deployment you provide your own values anyway.